sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 3.2.0 (2023-09-28)
```

Comment:
.NET SDK has its own version format, please update it back to `1.1.0`

Reply:
Regenerated the code again and update change log and cs project version as suggested.